swift/StableDiffusion/pipeline/StableDiffusionPipeline.swift

```diff
@@ -160,6 +162,7 @@ public struct StableDiffusionPipeline: ResourceManaging {
             switch config.schedulerType {
             case .pndmScheduler: return PNDMScheduler(stepCount: config.stepCount)
             case .dpmSolverMultistepScheduler: return DPMSolverMultistepScheduler(stepCount: config.stepCount)
+            case .eulerAncestralDiscreteScheduler: return EulerAncestralDiscreteScheduler(randomSource: randomSource(from: config.rngType, seed: config.seed))
```

Comment:
The new scheduler does not take `config.stepCount` as an argument, is this expected?

Reply:
Related: Could you please register this scheduler in the [CLI enum](https://github.com/apple/ml-stable-diffusion/blob/main/swift/StableDiffusionCLI/main.swift#L281) as well?

Reply:
No, this is not expected. Seems like I missed this. Thanks for pointing out.

I'll register it in CLI enum.

Reply:
I've made the changes, but still couldn't seem to generate a proper image. Currently still looking into what's the issue.